src/server/models/users.model.js

```diff
@@ -70,7 +70,12 @@ let userSchema = new Schema({
   education: [educationSchema],
   experience: [experienceSchema],
   work: [experienceSchema],
-  career: [careerSchema],
+  career: {
+    area: String,
```

Comment:
I think it makes sense to keep these as specific enum type (together with meta data to represent the string value). That'll save some "string comparisons" later when filtering etc. 
But yeah, totally optional, we could tackle it later.